lib/action_cable/test_helper.rb

```diff
@@ -127,5 +131,21 @@ def pubsub_adapter # :nodoc:
       def broadcasts_size(channel) # :nodoc:
         broadcasts(channel).size
       end
+
+      def stream(target, channel = nil)
+        return target if target.is_a?(String)
+
+        channel ||= @subscription
+        if channel && channel.respond_to?(:channel_name)
```

Comment:
Let's avoid additional _raising_ method and write smth like:

```ruby
...
raise CHANNEL_NOT_FOUND unless channel && channel.respond_to?(:channel_name)
channel.broadcasting_for(...)
```

And move the message itself into a constant.

Reply:
@palkan I've created separate error class for this. Not sure if that's what you were asking, though